modules/system/models/PluginVersion.php

```diff
@@ -116,6 +117,40 @@ public function afterFetch()
         }
     }
 
+    /**
+     * After the model is saved
+     */
+    public function afterSave()
+    {
+        if ($this->getOriginal('is_disabled') === 0 && $this->is_disabled === 1) {
```

Comment:
These comparison checks should probably be loose type comparisons, just in case some databases don't return those values as integers but as actual booleans.
```suggestion
        if (!$this->getOriginal('is_disabled') && $this->is_disabled) {
```